templates/docs/layouts/documentation.md

```diff
@@ -0,0 +1,86 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Documentation | Layouts
+---
+
+## Documentation layout
+
+<hr>
+
+### Structure
+
+The documentation page layout can be built using the Vanilla grid classes and common components. It consists of 3 horizontal areas that span the entire fixed width of the grid: header, content, footer.
+
+![Documentation layout structure](https://assets.ubuntu.com/v1/cdf09557-Documentation+layout%401x.svg)
+
+At the large breakpoint, the content area is further divided into an aside (3 columns) and a main content area (9 columns).
+
+At smaller breakpoints, the aside is moved offscreen and shown / hidden using a toggle.
+
+#### Header
+
+The header with the main navigation, built with the [navigation component](/docs/patterns/navigation#global-navigation).
+
+Style and contents of the documentation main navigation should be consistent with rest of the site.
```

Comment:
Not sure what this is meant to highlight?

Reply:
Just to make sure that the main navigation is the same across the site. We had the cases (Vanilla was one of those), where main nav was inconsistent between docs and rest of the page.